doc/classes/OptionButton.xml

```diff
@@ -170,6 +177,14 @@
 				Sets the icon of the item at index [param idx].
 			</description>
 		</method>
+		<method name="set_item_icon_max_width">
+			<return type="void" />
+			<param index="0" name="idx" type="int" />
+			<param index="1" name="width" type="int" />
+			<description>
+				Sets the maximum allowed width of the icon for the item at the given [param idx]. The height is adjusted according to the icon's ratio.
```

Comment:
It might be nice to link to `[method PopupMenu.set_item_icon_max_width]` here to have cross coverage.

Reply:
🤔 You're probably right, but if that's the case we should probably add it to all the `get_item_*` methods yeah? I feel like it would be weird if this was the only one that cross linked...

Reply:
tbh this method is not even necessary. You can use `set_icon_max_width()` in the PopupMenu, it will have the same effect.